user_guide.md

```diff
@@ -550,6 +551,14 @@ tmpfs                                                           15444244       0
 
 ## Troubleshooting
 
+### TensorFlow and AVX
+There are some instances where you may encounter a TensorFlow-related Python installation or a pod launch issue that results in a SIGILL (illegal instruction core dump). Kubeflow uses the pre-built binaries from the TensorFlow project which, beginning with version 1.6, are compiled to make use of the [AVX](https://en.wikipedia.org/wiki/Advanced_Vector_Extensions) CPU instruction. This is a recent feature and your CPU might not support it. Check the host environment for your node to determine whether it has this support:
```

Comment:
SIGILL typo

Reply:
What's the typo?

Reply:
My bad. I thought it was supposed to be SIGKILL

Reply:
👍